core/src/main/java/org/springframework/ldap/odm/core/impl/ObjectMetaData.java

```diff
@@ -18,9 +18,11 @@
 
```

Comment:
Will you please update the copyright year for each one of the files you modify? The end year should be changed to 2022.

Reply:
Done :)